<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WingMenuConfiguration</class>
 <widget class="QDialog" name="WingMenuConfiguration">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WingMenu settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Panel Button</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="iconCB">
            <property name="text">
             <string>Show icon:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="iconLE"/>
             </item>
             <item>
              <widget class="QPushButton" name="iconPB">
               <property name="text">
                <string>Browse</string>
               </property>
               <property name="icon">
                <iconset theme="folder"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="textCB">
            <property name="text">
             <string>Show text:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="textLE"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Menu file</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="menuFileLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="menuFilePB">
            <property name="text">
             <string>Browse</string>
            </property>
            <property name="icon">
             <iconset theme="folder"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>D-Bus Activation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="dBusLE">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Keyboard Shortcut</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="ShortcutSelector" name="shortcutEd">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Click the button to record shortcut:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="categoryLeftCB">
            <property name="text">
             <string>Categories on the Left</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="searchBottomCB">
            <property name="text">
             <string>Search on Bottom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="sidebarLeftCB">
            <property name="text">
             <string>Sidebar on the Left</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="reverseSidebarCB">
            <property name="text">
             <string>Reverse Sidebar</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Applications Layout:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="appLayoutCombo">
            <item>
             <property name="text">
              <string>List (Name and Description)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>List (Name only)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Icons</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="switchOnHoverCB">
            <property name="text">
             <string>Switch Category on hover</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="hoverDelaySB">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="prefix">
             <string>Delay: </string>
            </property>
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>50</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="askFavoriteRemoveCB">
            <property name="text">
             <string>Ask before removing Favorite</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>163</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="LeaveActionsTab">
      <attribute name="title">
       <string>Leave Actions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="customizeLeaveGB">
         <property name="title">
          <string>Customize Leave Actions</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="loadFromMenuPB">
               <property name="text">
                <string>Load from menu file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addDesktopFilePB">
               <property name="text">
                <string>Add .desktop file</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="locationLB">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>The order displayed here ignores the option &quot;Reverse sidebar&quot;.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QListView" name="leaveActionsView">
               <property name="editTriggers">
                <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>true</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
               </property>
               <property name="defaultDropAction">
                <enum>Qt::DropAction::IgnoreAction</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="newActionPB">
                  <property name="text">
                   <string>New</string>
                  </property>
                  <property name="icon">
                   <iconset theme="document-new"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="editActionPB">
                  <property name="text">
                   <string>Edit</string>
                  </property>
                  <property name="icon">
                   <iconset theme="document-edit"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="upActionPB">
                  <property name="text">
                   <string>Up</string>
                  </property>
                  <property name="icon">
                   <iconset theme="go-up"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="downActionPB">
                  <property name="text">
                   <string>Down</string>
                  </property>
                  <property name="icon">
                   <iconset theme="go-down"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="removeActionPB">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                  <property name="icon">
                   <iconset theme="edit-delete"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutSelector</class>
   <extends>QToolButton</extends>
   <header location="global">LXQtGlobalKeysUi/ShortcutSelector</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>iconCB</tabstop>
  <tabstop>iconLE</tabstop>
  <tabstop>iconPB</tabstop>
  <tabstop>textCB</tabstop>
  <tabstop>textLE</tabstop>
  <tabstop>menuFileLE</tabstop>
  <tabstop>menuFilePB</tabstop>
  <tabstop>shortcutEd</tabstop>
  <tabstop>categoryLeftCB</tabstop>
  <tabstop>searchBottomCB</tabstop>
  <tabstop>sidebarLeftCB</tabstop>
  <tabstop>reverseSidebarCB</tabstop>
  <tabstop>appLayoutCombo</tabstop>
  <tabstop>switchOnHoverCB</tabstop>
  <tabstop>hoverDelaySB</tabstop>
  <tabstop>askFavoriteRemoveCB</tabstop>
  <tabstop>customizeLeaveGB</tabstop>
  <tabstop>loadFromMenuPB</tabstop>
  <tabstop>addDesktopFilePB</tabstop>
  <tabstop>leaveActionsView</tabstop>
  <tabstop>newActionPB</tabstop>
  <tabstop>editActionPB</tabstop>
  <tabstop>upActionPB</tabstop>
  <tabstop>downActionPB</tabstop>
  <tabstop>removeActionPB</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switchOnHoverCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>hoverDelaySB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
